source/js/components/currency-selector.js

```diff
@@ -96,10 +99,26 @@ class CurrencySelect {
       .join("");
 
     var otherAmountString = window.gettext("Other amount");
+    var minimumOtherAmountString = window.gettext("Minimum amount is");
+    var invalidAmountString = window.gettext("Invalid amount entered.");
     container.insertAdjacentHTML(
       "beforeend",
-      `<div class='donation-amount donation-amount--two-col donation-amount--other'><input type='radio' class='donation-amount__radio' name='amount' value='other' id='${type}-other' autocomplete='off' data-other-amount-radio><label for='${type}-other' class='donation-amount__label' data-currency>$</label><input type='number' class='donation-amount__input' id='${type}-other-input' placeholder='${otherAmountString}' data-other-amount min="${minAmount}" max="10000000"></div>`
+      `
+      <div class='donation-amount--two-col'>
+        <div class='donation-amount donation-amount--other'>
+          <input type='radio' class='donation-amount__radio ${type}-donation-radio' name='amount' value='other' id='${type}-other' autocomplete='off' data-other-amount-radio>
+          <label for='${type}-other' class='donation-amount__label' data-currency>$</label>
+          <input type='number' class='donation-amount__input' id='${type}-other-input' placeholder='${otherAmountString}' data-other-amount min="${minAmount}" max="10000000">
+        </div>
+        <p class='minimum'>
+          ${minimumOtherAmountString} ${symbol}${minAmount}
```

Comment:
We will need to adapt the approach here as well, I think you can use the `formattedValue` variable directly (no need to pass the symbol explicitly), combined with [string interpolation](https://docs.djangoproject.com/en/4.0/topics/i18n/translation/#interpolate)?

Reply:
Thanks for the suggestion Theo! using the formatter that was used to create formattedValue works great, now when the user selects a currency, the symbol matches the format on the buttons. Instead of directly injecting the symbol.